fhir-converter/src/routes.js

```diff
@@ -869,6 +876,9 @@ module.exports = function (app) {
         if (!retInvalidAccess) {
           delete resultMessage["invalidAccess"];
         }
+        console.log(
+          `[patient ${patientId}] Took ${duration}ms / status ${result.status} to process file ${fileName}`
```

Comment:
hypernit: wording here is a bit weird -> "Took 100ms / status 200 to process file blah.xml"

maybe: "Took 100ms to process file blah.xml - with status 200"